hkmc2/jvm/src/test/scala/hkmc2/JSBackendDiffMaker.scala

```diff
@@ -13,6 +18,7 @@ abstract class JSBackendDiffMaker extends MLsDiffMaker:
   val sjs = NullaryCommand("sjs")
   val showRepl = NullaryCommand("showRepl")
   val silent = NullaryCommand("silent")
+  val jsSanityCheck = NullaryCommand("jsSanityCheck")
```

Comment:
Pls rename to
```suggestion
  val noSanityChecks = NullaryCommand("noSanityChecks")
```
so that sanity checks are the default.